draft-kleidl-digest-fields-problem-types.md

```diff
@@ -78,25 +78,71 @@ interpreted as described in {{PROBLEM}}.
 
 ## Unsupported Hashing Algorithm
 
-This section defines the "https://iana.org/assignments/http-problem-types#unsupported-hashing-algorithm" problem type. A server MAY use this problem type when responding to a request, whose integrity or integrity preference fields reference a hashing algorithm that the server can not or does not want to support for this request, and if the server wants to indicate this problem to the sender.
+This section defines the "https://iana.org/assignments/http-problem-types#unsupported-hashing-algorithm" problem type.
+A server MAY use this problem type if it wants to communicate to the client that
+none of the hashing algorithms referenced in the integrity or integrity preference fields present in the request,
```

Comment:
I agree. This is a tricky point, because the server is not required to process all algorithms.

We need to decide whether this problem-type applies to just one of the algorithms, or to all of the algorithms.

Reply:
Opened https://github.com/tus/draft-digest-fields-problem-types/issues/18 to track this question.